quarkus/config-api/src/main/java/org/keycloak/config/DatabaseOptions.java

```diff
@@ -10,6 +10,13 @@ public class DatabaseOptions {
             .buildTime(true)
             .build();
 
+    public static final Option<Boolean> DB_LOG_HIBERNATE_STATISTICS = new OptionBuilder<>("db-log-hibernate-statistics", Boolean.class)
```

Comment:
I would drop `hibernate` from the CLI name

```suggestion
    public static final Option<Boolean> DB_LOG_HIBERNATE_STATISTICS = new OptionBuilder<>("db-log-statistics", Boolean.class)
```